front/src/modules/ui/layout/page-bar/components/PageBar.tsx

```diff
@@ -95,7 +95,7 @@ export function PageBar({
         <StyledLeftContainer>
           {!isNavbarOpened && (
             <StyledTopBarButtonContainer>
-              <NavCollapseButton direction="right" />
+              <NavCollapseButton direction="right" isVisible={true} />
```

Comment:
```suggestion
              <NavCollapseButton direction="right"/>
```

Reply:
I Changed all requested proposals for naming  but for this, it will throw an error Property `hide` is missing